quipucords/scanner/network/tests_network_connect.py

```diff
@@ -1,605 +1,605 @@
-        mock_run.side_effect = NetworkCancelException()
-        scanner = ConnectTaskRunner(self.scan_job3, self.scan_task3)
-        _, scan_result = scanner.run(Value("i", ScanJob.JOB_RUN))
-        self.assertEqual(scan_result, ScanTask.CANCELED)
-
-    @patch("scanner.network.connect.ConnectTaskRunner.run_with_result_store")
-    def test_pause_connect(self, mock_run):
-        """Test pause of connect."""
-        # Test cancel at _connect level
-        serializer = SourceSerializer(self.source3)
-        source = serializer.data
-        hosts = source["hosts"]
-        connection_port = source["port"]
-        with self.assertRaises(NetworkPauseException):
-            _connect(
-                Value("i", ScanJob.JOB_TERMINATE_PAUSE),
-                self.scan_task,
-                hosts,
-                Mock(),
-                self.cred,
-                connection_port,
-                self.concurrency,
-            )
-        # Test cancel at run() level
-        mock_run.side_effect = NetworkPauseException()
-        scanner = ConnectTaskRunner(self.scan_job3, self.scan_task3)
-        _, scan_result = scanner.run(Value("i", ScanJob.JOB_RUN))
-        self.assertEqual(scan_result, ScanTask.PAUSED)
-
-    def test_run_manager_interupt(self):
-        """Test manager interupt for run method."""
-        scanner = ConnectTaskRunner(self.scan_job, self.scan_task)
-        conn_dict = scanner.run(Value("i", ScanJob.JOB_TERMINATE_CANCEL))
-        self.assertEqual(conn_dict[1], ScanTask.CANCELED)
-
-    @patch("scanner.network.connect.ConnectTaskRunner.run_with_result_store")
-    def test_run_success_return_connect(self, mock_run):
-        """Test pause of connect."""
-        # Test cancel at run() level
-        mock_run.side_effect = [[None, ScanTask.COMPLETED]]
-        scanner = ConnectTaskRunner(self.scan_job3, self.scan_task3)
-        _, scan_result = scanner.run(Value("i", ScanJob.JOB_RUN))
-        self.assertEqual(scan_result, ScanTask.COMPLETED)
-
-    @patch("scanner.network.connect._connect")
-    def test_connect_exception(self, mock_run):
-        """Test pause of connect."""
-        # Test cancel at run() level
-        mock_run.side_effect = AnsibleRunnerException("fail")
-        scanner = ConnectTaskRunner(self.scan_job3, self.scan_task3)
-        _, scan_result = scanner.run(Value("i", ScanJob.JOB_RUN))
-        self.assertEqual(scan_result, ScanTask.FAILED)
-
-    @patch("ansible_runner.run")
-    def test_empty_hosts(self, mock_run):
-        """Test running a connect scan with mocked connection."""
-        mock_run.return_value.status = "successful"
-        scanner = ConnectTaskRunner(self.scan_job, self.scan_task)
-        result_store = MockResultStore([])
-        _, result = scanner.run_with_result_store(
-            Value("i", ScanJob.JOB_RUN), result_store
-        )
-        self.assertEqual(result, ScanTask.COMPLETED)
+"""Test the discovery scanner capabilities."""
```

Comment:
Why did this whole file change? Line endings?

Reply:
the only source file in the repo with DOS line endings, https://github.com/quipucords/quipucords/pull/2276/commits/14dbcfdea687540242d28df7f7f536cff7a0259d.    was showing diffs with ^M at end.  ye-old dos2unix to the rescue.

Reply:
Ugh. DOS line endings. 🤢 

Good riddance! Thank you! 👏